src/globals/scss/_feature-flags.scss

```diff
@@ -3,7 +3,9 @@
 //-------------------------
 
 // Initialize our feature flag map with default values
-$feature-flags: () !default;
+$feature-flags: (
+  components: false,
```

Comment:
Do we want to do `component-experimental` or `experimental-components` to reflect what this flag does?

Reply:
Thats a good idea 😄

Reply:
Maybe we should change it to
```
$experimental-flag (
components: true;
)
```
@joshblack thoughts on this? It's not entirely clear what it's doing right now.

Reply:
What about `$experimental-feature-flags` 

I think we should have experimental in there somewhere since thats what consumers will see

Reply:
Thanks @alisonjoseph and @joshblack for your thoughts! Important points from both, the feature flags are for roping things off feature-by-feature basis, and we need to be clear on what incoming features they are. Another thing I’ve been thinking about is releasing roped-off feature should be as simple as flipping the flag.

All that said - Though I brought up “experimental” word, I’m now thinking that the word itself is too general, especially considering another initiative of experimental theme coming after what we are currently working on - Which led me to:

```scss
$feature-flags: (
  components-x: false,
}
```

Though it’s lame. Thoughts on any better words? Thanks!

Reply:
If we change it to `components-x` then this part of the code would seems strange. 
```
@include exports('checkbox') {
  @include stable('components') {
    @include checkbox;
  }

  @include experimental('components') {
    @include checkbox--experimental;
  }
}
```

Reply:
@alisonjoseph Great point!! What about:

```scss
@include exports('checkbox') {
  @include disabled('components-x') {
    @include checkbox;
  }

  @include enabled('components-x') {
    @include checkbox--experimental;
  }
}
```

BTW as I have signed off this PR, this discussion shouldn’t block this PR to go forward.

Reply:
We could discuss at the weekly code review tomorrow with the whole dev team also 😅